control/control.rnc

```diff
@@ -67,6 +67,7 @@ textdomain = element textdomain { text }
 
 globals_elements =
     incomplete_translation_treshold
+    | product_name
```

Comment:
I'd put it into the `<software>` section ([here](https://github.com/yast/yast-installation-control/blob/51acc93b75cbbd2e25db98212bd178893d2ab138/control/control.rnc#L287)) instead of the `<globals>` as it is related to package selection.

Also I'd prefer a more descriptive name like `<select_product>` or something like that.